crates/protocol/interop/src/depset.rs

```diff
@@ -6,20 +6,7 @@ use kona_registry::HashMap;
 #[derive(Debug, Clone, PartialEq, Eq)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[cfg_attr(feature = "serde", serde(rename_all = "camelCase"))]
-pub struct ChainDependency {
-    /// The unique short identifier for this chain.
-    pub chain_index: u32,
-
-    /// Timestamp (in seconds) when the chain becomes part of the dependency set.
-    /// This is the minimum timestamp of the inclusion of an executing message that can be
-    /// considered valid.
-    pub activation_time: u64,
-
-    /// Lower bound timestamp (in seconds) for data storage and accessibility.
-    /// This is the minimum timestamp of an initiating message that is required to be
-    /// accessible to other chains. This is set to 0 when all data since genesis is executable.
-    pub history_min_time: u64,
-}
+pub struct ChainDependency {}
```

Comment:
```suggestion
pub struct ChainDependency;
```
preferred syntax for ZST